docs/site/data/docs/v0.11-toc.yml

```diff
@@ -0,0 +1,274 @@
+toc:
```

Comment:
@joshrosso TODO verification that this is consistent with `main`.

Reply:
Yup this is good to go